ec/src/lib.rs

```diff
@@ -245,9 +245,18 @@ pub trait AffineCurve:
     + for<'a> core::iter::Sum<&'a Self>
     + From<<Self as AffineCurve>::Projective>
 {
+    /// Cofactor of the subgroup defined by `order / prime order of subgroup`
     const COFACTOR: &'static [u64];
+
+    /// Set of scalars in a prime field with efficient square root
```

Comment:
Do you think it is better to say "type of scalars" instead of "set of scalars"? Just because Rust has "set", and I am afraid of confusion.

```suggestion
    /// Type of scalars in a prime field with efficient square root
```